decidim-blogs/app/views/decidim/blogs/posts/show.html.erb

```diff
@@ -15,47 +15,81 @@
   )
 %>
 
-<div class="row column view-header">
-  <div class="m-bottom">
-    <%= link_to :posts, class: "small hollow" do %>
-      <%= icon "chevron-left", class: "icon--small", role: "img", "aria-hidden": true %>
-      <%= t(".back") %>
-    <% end %>
-  </div>
-  <h2 class="heading2"><%= translated_attribute post.title %></h2>
-
-  <% post_presenter = Decidim::Blogs::PostPresenter.new(post) %>
-  <%= cell "decidim/author", post_presenter.author, from: post %>
-</div>
-<div class="row">
-  <% if show_endorsements_card? %>
-      <div class="columns section view-side mediumlarge-4 mediumlarge-push-8 large-3 large-push-9">
-        <div class="card">
-          <div class="card__content">
-            <div class="row collapse buttons__row">
-              <% if endorsements_enabled? %>
-                <div class="column small-9 collapse">
-                  <%= endorsement_buttons_cell(post) %>
-                </div>
-              <% end %>
-              <div class="column collapse <%= endorsements_enabled? ? "small-3" : "" %>">
-                <%= link_to "#comments", class: "button small compact hollow button--nomargin expanded" do %>
-                    <%= icon "comment-square", class: "icon--small", aria_label: t(".comments"), role: "img" %> <%= post.comments_count %>
-                <% end %>
-              </div>
-            </div>
-            <br>
-            <%= follow_button_for(post) %>
-          </div>
-        </div>
-      </div>
+<% content_for :css_content do %>
+  <%= stylesheet_pack_tag "decidim_blogs", media: "all" %>
+<% end %>
+
+<%= render layout: "layouts/decidim/shared/layout_center" do %>
+
+  <h1 class="h1 decorator my-12"><%= translated_attribute post.title %></h1>
+
+  <% if post.photo.present? %>
+    <div class="blog__image grow">
+      <%# REDESIGN_PENDING: currently post.photo has no url attribute. Uncomment next line then %>
+      <%#= image_tag post.photo.url %>
+    </div>
   <% end %>
-  <div class="columns mediumlarge-8 <%= "mediumlarge-pull-4" if show_endorsements_card? %>">
-    <div class="section">
+
+  <div class="blog__container">
+    <% post_presenter = Decidim::Blogs::PostPresenter.new(post) %>
+    <%= cell "decidim/author", post_presenter.author, from: post, context_actions: [:date], layout: :compact %>
+
+    <div class="blog__text">
       <%= decidim_sanitize_editor translated_attribute post.body %>
     </div>
-    <%= cell "decidim/endorsers_list", post %>
+
+    <div class="blog__actions">
+      <div class="blog__actions-left">
+        <% if endorsements_enabled? %>
+          <div id="endorsers_list-trigger" class="cursor-pointer" data-toggler=".is-active" data-toggle="endorsers_list-trigger endorsers_list" role="button">
```

Comment:
The `data-toggle/data-toggler` attributes are part of the current [Foundation Toggler](https://get.foundation/sites/docs/toggler.html). The pros are that this script make the elements accessible, the cons are that Foundation is required. As an alternative, I come across [this lib](https://edenspiekermann.github.io/a11y-toggle/) but I was not fully convinced.

In any case, I think it's a better approach to do this large changes (multiple views affected) in late phases of the redesign, to make them all in once.

Do you prefer to kep the foundation library or replace with another one? Any other thoughts? cc/ @ahukkanen @andreslucena

Reply:
@Crashillo I'd try to get rid of Foundation completely, so always when there's a possibility for a refactor out of Foundation, I vote for the refactor. The main reason is that it's on the roadmap to get rid of jQuery some time in the future, so getting rid of Foundation also makes that easier along the line.

That said, I quickly glanced through the suggested replacement library and otherwise it seems good but only missing thing from Foundation is that it doesn't emit any events when changes happen with the toggled element. This would make it hard to listen to the changes and it would have to be done through MutationObserver or similar techniques which is not optimal.

And this is not only a "rant" about missing features, I actually needed this feature in core when implementing #9582.